src/app/public/modules/description-list/description-list.service.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  BehaviorSubject
+} from 'rxjs';
+
+/**
+ * @internal
+ */
+@Injectable()
+export class SkyDescriptionListService {
+
+  public defaultDescription = new BehaviorSubject<string>('');
```

Comment:
We should probably complete this observable in an `ngOnDestroy`. Also, I would change this to a getter/setter so that subscribers can't affect/complete the observable themselves.
```
public get defaultDescription(): Observable<string> {
  return this._defaultDescription.asObservable();
}

public set defaultDescription(value: string) {
  this._defaultDescription.next(value);
}
```

Reply:
Done.